.travis.yml

```diff
@@ -5,13 +5,14 @@ matrix:
       env: TOX_ENV=pep8
     - python: "3.4"
       env: TOX_ENV=py34
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y build-essential libcap-dev
 install:
   - pip install tox
-  - pip install python-coveralls
+  - if [[ $TOX_ENV == py34 ]]; then
+      pip install coveralls;
+    fi
```

Comment:
First, we only run coveralls if we're running our tests in the py34 environment (the pep8 environment doesn't have any coverage!).

Second, we're installing the `coveralls` package instead of the `python-coveralls` package. `python-coveralls` hasn't been working for me locally.